sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -59,34 +59,35 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
   };
 
   /**
-   * @brief HTTP request retry options.
+   * @brief The set of options that can be specified to influence how retry attempts are made, and a
+   * failure is eligible to be retried.
    * @note See https://azure.github.io/azure-sdk/general_azurecore.html#retry-policy.
    *
    */
   struct RetryOptions final
   {
     /**
-     * @brief Maximum number of attempts to retry.
+     * @brief The maximum number of retry attempts before giving up.
      *
      */
     int32_t MaxRetries = 3;
 
     /**
-     * @brief Mimimum amount of milliseconds between retry attempts.
+     * @brief The minimum permissible delay between retry attempts.
      * @note See https://en.cppreference.com/w/cpp/chrono/duration.
      *
      */
     std::chrono::milliseconds RetryDelay = std::chrono::seconds(4);
 
     /**
-     * @brief Mimimum amount of milliseconds between retry attempts.
+     * @brief The maximum permissible delay between retry attempts.
      * @note See https://en.cppreference.com/w/cpp/chrono/duration.
      *
      */
     std::chrono::milliseconds MaxRetryDelay = std::chrono::minutes(2);
 
     /**
-     * @brief HTTP status codes to retry on.
+     * @brief The HTTP status codes that indicate the operation should be retried.
```

Comment:
```suggestion
     * @brief The HTTP status codes that indicate when an operation should be retried.
```

Reply:
I think "that the operation should be retried" would be even better, what do you think?

Reply:
I thought the operation is only retried on certain status codes, no?

Reply:
i.e. "when" sounds like some point in time, a little bit. But, of course, not exclusively, it sounds as condition as well, and it is not confusing anyways. I went with your suggestion.

Reply:
Also, a nit regarding your suggestion Anton, "... that indicate that ..." makes it slightly difficult to parse/read, due to the repetition.

Reply:
The operation gets retried on status codes, your understanding is correct.
I am just thinking between  "to indicate that" vs "to indicate when". But either way put it is understandable.
I agree that repetition so close does not read well. So let's go with "when" (I already committed it).